sdk/src/main/kotlin/io/rover/sdk/ui/blocks/poll/image/ViewImagePoll.kt

```diff
@@ -165,6 +166,7 @@ internal class ViewImagePoll(override val view: LinearLayout) :
     private fun bindQuestion(imagePoll: ImagePoll) {
         questionView.run {
             text = imagePoll.question.rawValue
+            contentDescription = "This is a poll. ${imagePoll.question.rawValue}. Swipe to hear options"
```

Comment:
I'm pretty sure we shouldn't give specific directions to the user on how to use the a11y access method on their device.  The a11y APIs provide *semantic* access to many different means of using the software, which may or may not involve swiping.  That string assumes the user is using TalkBack here, but they may be using any number of different accessibility modes depending on their personal requirements. Some of which may not even exist yet!

From what I understand a11y users care about concision and brevity.

So I suggest deleting the "Swipe to hear options" phrase entirely, and making it all a single concise sentence.  Sean used the following as the accessibility label for the question view on iOS, and I think it's about right (non-verbose and concise):

    "Poll with $numberOfOptions options: $questionText"